shared/constants/fs.js

```diff
@@ -113,3 +126,9 @@ export const getItemStyles = (
       return isPublic ? itemStylesPublicUnknown : itemStylesPrivateUnknown
   }
 }
+
+export const makeDownloadKey = (path: Types.Path, localPath: string) =>
+  `download:${Types.pathToString(path)}:${localPath}`
+
+export const downloadFilePathFromPath = (p: Types.Path): Promise<Types.LocalPath> =>
+  downloadFilePath(Types.getPathName(p))
```

Comment:
This is probably fine but it looks like it still puts it in the temp directory. I found [this package](https://github.com/juliangruber/downloads-folder), for instance, which uses `${process.env.USERPROFILE}/Downloads` on Windows. NBD, fine to ship it like this.

Reply:
Thanks! But I think `downloadFilePath` already downloads to the Downloads folder, and doesn't use the temp directory at all. Here's where it's from: https://github.com/keybase/client/blob/406972ce2a6df37cd8bf8d7054de7bca6850d379/shared/util/file.desktop.js#L32

Sorry I forgot to follow-up to my own question above!